### PR TITLE
feat: footer

### PR DESCRIPTION
**Pull request by Shivanshsharma01**
Added footer and readme and some refactoring

image

### CODE REVIEW

File: Page/product/product.html [hunk 1]
```diff
@@ -257,11 +259,29 @@
...
-
             </div>
         </div>
     </main>
+    <footer class="footer ">
+
+        <div class="footer-header">Made with 💖 by Shivansh Sharma</div>
+        <ul class="footer-list">
+            <li class="list-item-inline">
+                <a class="links " href="https://github.com/Shivanshsharma01" target="blank">Githhub <i
```
Comment (shashidwivedi) [inline Page/product/product.html]: You can use target="_blank" here

File: Page/signup/signup.html [hunk 2]
```diff
@@ -14,46 +15,67 @@
...
             <a class="cart-icon" href="/page/wishlist/wishlist.html"><i class="fa-regular fa-heart"></i></a>
             <a class="cart-icon" href="/page/cart/cart.html"><i class="fa-solid fa-cart-shopping-fast"></i></a>
         </nav>
+    </header>
+
+    <main>
+        <div class="log-container">
+            <div class="log-heading">Sign UP</div>
+            <div class="log-input">
+                <div class="input-group ">
```
Comment (shashidwivedi) [inline Page/signup/signup.html:30]: There is whitespace after a single class name and ending quote. Will suggest you to use prettier or some other formatter

**Review by shashidwivedi — APPROVED** [review]
Rest looks fine, can be merged

File: Page/wishlist/wishlist.html [hunk 3]
```diff
@@ -59,76 +60,96 @@
...
+
                 </div>
 
-            <div class="card .price-card">
-                <div class="product-image">
-                    <img src="https://images.unsplash.com/photo-1612724189298-89d36b10b26d?ixlib=rb-1.2.1&ixid=MnwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8&auto=format&fit=crop&w=1025&q=80"
-                        alt="Image" />
-                    <div class="card-badge-dismiss">
-                        <i class="fa-regular fa-circle-xmark"></i>
+                <div class="card .price-card">
```
Comment (prachi-sahani) [inline Page/wishlist/wishlist.html]: class name with a dot(.price-card)? I suppose that is done by mistake

File: css/footer.css [hunk 4]
```diff
@@ -0,0 +1,40 @@
...
+    margin-top: 5rem;
+    position: relative;
+}
+
+.footer-list {
+    margin-top: 1rem;
+}
+
+.footer ul {
+    padding-inline-start: 0px;
```
Comment (prachi-sahani) [inline css/footer.css:16]: you need not mention the unit(only 0 is sufficient)

File: css/footer.css [hunk 5]
```diff
@@ -0,0 +1,40 @@
...
+
+.links {
+    text-decoration: none;
+    padding: 0.5rem;
+    color: var(--nav-logo);
+}
+
+.list-item-inline {
+    display: inline;
+    padding: 0rem 0.5rem;
```
Comment (prachi-sahani) [inline css/footer.css:31]: you need not mention the unit(only 0 is sufficient)

File: Page/product/product.html [hunk 6]
```diff
@@ -257,11 +259,29 @@
                     <button class="btn btn-success"><i class="fa-regular fa-heart"></i>
                         WISHLIST</button>
                 </div>
-
-
             </div>
         </div>
     </main>
+    <footer class="footer ">
```
Comment (shuklahimanshu81) [inline Page/product/product.html:265]: Left space after class name

**Review by prachi-sahani — COMMENTED** [review]
Apart from the mentioned points, the code is good to merge!

File: Page/wishlist/wishlist.html [hunk 7]
```diff
@@ -59,76 +60,96 @@
...
             </div>
         </div>
-        </div>
     </main>
+    <footer class="footer ">
+
+        <div class="footer-header">Made with 💖 by Shivansh Sharma</div>
+        <ul class="footer-list">
+            <li class="list-item-inline">
+                <a class="links " href="https://github.com/Shivanshsharma01" target="blank">Githhub <i
```
Comment (shuklahimanshu81) [inline Page/wishlist/wishlist.html]: It should be target ="_blank"

**Review by shuklahimanshu81 — APPROVED** [review]
Over many places you forgot to remove space  after class name. And also underscore before blank. Everything else is good. Good to go for merge .

**Review by shivangisharma01 — APPROVED** [review]
Well structured and variable names  are also good.

**Review by shivangisharma01 — APPROVED** [review]

**Comment by omjadhav85** [comment]
Apart from the comments already made...looks good to me! Great work!

**Pull request merged**